src/main/java/de/tum/in/www1/artemis/web/rest/ModelingExerciseResource.java

```diff
@@ -213,6 +213,7 @@ public ModelingExerciseResource(ModelingExerciseRepository modelingExerciseRepos
     @PreAuthorize("hasAnyRole('TA', 'INSTRUCTOR', 'ADMIN')")
     public ResponseEntity<ModelingExercise> getModelingExercise(@PathVariable Long id) {
         log.debug("REST request to get ModelingExercise : {}", id);
+        // TODO CZ: provide separate endpoint GET /modeling-exercises/{id}/withExampleSubmissions and load exercise without example submissions here
```

Comment:
should we leave todos like that in the code?

Reply:
it is not ideal, but some of these TODOs describe lower priority tasks that we want to finish in the future, so better merging these TODOs now than loosing a lot of time.